src/Hooks.js

```diff
@@ -0,0 +1,20 @@
+export class Hooks {
+  static passThroughHooks = new Set([
+    'preprocess',
+    'postprocess'
+  ]);
```

Comment:
Currently both `preprocess` and `postprocess` are "passthrough" hooks, which means the return value of one function gets passed to the next hook.

In the future I could see us adding some hooks that would not pass the return to the next hook. For example the way we call renderer or tokenizer methods is not pass through. The first tokenizer is called and if it returns something other than  `false` we don't call the next tokenizer of the same name at all.